pkg/apis/componentconfig/edgecore/v1alpha1/types.go

```diff
@@ -235,6 +235,9 @@ type Edged struct {
 	// EnableMetrics indicates whether enable the metrics
 	// default true
 	EnableMetrics bool `json:"enableMetrics,omitempty"`
+	// CsiDriverName indicates the csi driver name used by edge node
+	// default "csi-hostpath"
+	CsiDriverName string `json:"csiDriverName,omitempty"`
```

Comment:
cannot be `omitempty`, we need a not-empty value here.

Reply:
And may I ask for more materials about application scenarios?

Reply:
The usable csi driver of edge node not just hostpath type, it may be lvm, nfs .etc.
We can not fixed as csi-hostpath in code.